### PR TITLE
Add lesson 26 notes

### PR DESCRIPTION
**Pull request by angelsvirkov**

### CODE REVIEW

File: lesson26/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Lesson 26 - Web Accessibility
+nav_order: 26
+---
+
+## Agenda
+
+1. Introduction to accessibility
```
Comment (impactmass) [inline lesson26/index.md:8]:
```suggestion
1. Introduction to Accessibility
```

File: lesson26/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+1. Classification of disabilities and assistive technologies
+1. Accessibility guidelines
+1. WAI-ARIA
+1. Accessibility checkers
+
+## 1. Introduction to accessibility
```
Comment (impactmass) [inline lesson26/index.md:14]: I'm not sure, but think it should be capitalized
```suggestion
## 1. Introduction to Accessibility
```

File: lesson26/index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Accessibility means that **any** user can use all the features and the content of a particular software, regardless of how the user accesses it, especially if they are physically or mentally impaired. In this section we are going to cover the main aspects of accessibility together with the biggest benefits it provides to you and to your users.
```
Comment (impactmass) [inline lesson26/index.md:16]: Hey, I do not understand how accessibility helps someone mentally impaired. Or maybe, what do you mean by "mentally impaired"? I went through the meaning written here https://www.collinsdictionary.com/amp/english/mental-impairment ... and don't get how accessibility helps it

Comment (angelsvirkov) [reply lesson26/index.md:16]: Oh good catch. I meant something else https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility#people_with_cognitive_impairments . I will adjust the wording

**Review by impactmass — APPROVED** [review]

**Pull request merged**